src/Compilers/Core/Portable/Compilation/SemanticModel.cs

```diff
@@ -96,6 +96,50 @@ internal IOperation GetOperationInternal(SyntaxNode node, CancellationToken canc
 
         protected abstract IOperation GetOperationCore(SyntaxNode node, CancellationToken cancellationToken);
 
+#pragma warning disable RS0026 // Do not add multiple public overloads with optional parameters
+        /// <summary>
+        /// Determines what type of conversion, if any, would be used if a given expression was
+        /// converted to a given type.  If isExplicitInSource is true, the conversion produced is
+        /// that which would be used if the conversion were done for a cast expression.
+        /// </summary>
+        public ConversionInfo ClassifyConversionInfo(SyntaxNode expression, ITypeSymbol destination, bool isExplicitInSource = false)
+            => CommonClassifyConversionInfo(expression, destination, isExplicitInSource);
+#pragma warning restore RS0026 // Do not add multiple public overloads with optional parameters
+
+        internal abstract ConversionInfo CommonClassifyConversionInfo(SyntaxNode expression, ITypeSymbol destination, bool isExplicitInSource);
+
+#pragma warning disable RS0026 // Do not add multiple public overloads with optional parameters
+        /// <summary>
+        /// Determines what type of conversion, if any, would be used if a given expression was
+        /// converted to a given type.  If isExplicitInSource is true, the conversion produced is
+        /// that which would be used if the conversion were done for a cast expression.
+        /// </summary>
+        /// <param name="position">The character position for determining the enclosing declaration
+        /// scope and accessibility.</param>
+        /// <param name="expression">The expression to classify. This expression does not need to be
+        /// present in the syntax tree associated with this object.</param>
+        /// <param name="destination">The type to attempt conversion to.</param>
+        /// <param name="isExplicitInSource">True if the conversion should be determined as for a cast expression.</param>
```

Comment:
That's a good question.  It looks like VB and C# just deviated here and VB never had support for this.  I'll add this to the list of things we need to figure out.